### PR TITLE
implemented solution

### PR DESCRIPTION
**Pull request by willwildred**
- [DEMO LINK](https://willwildred.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://willwildred.github.io/layout_product-cards/report/html_report/)

- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,58 @@
...
+        <ul class="stars">
+          <li class="stars__active-star"></li>
+
+          <li class="stars__active-star"></li>
+
+          <li class="stars__active-star"></li>
+
+          <li class="stars__active-star"></li>
+
+          <li class="stars__unactive-star"></li>
```
Comment (Esceype) [inline src/index.html:41]: If you have repeated tags, each of which occupies one line, i.e. closes in the same line where it opened, then you can not create an empty line between such tags
```suggestion
        <ul class="stars">
          <li class="stars__active-star"></li>
          <li class="stars__active-star"></li>
          <li class="stars__active-star"></li>
          <li class="stars__active-star"></li>
          <li class="stars__unactive-star"></li>
```

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,124 @@
...
+    background-position: center;
+  }
+
+  &__unactive-star {
+    height: 16px;
+    width: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```
Comment (Esceype) [inline src/styles/index.scss:123]: It`s wrong, according to the previous task, the stars block should have a modifier for the block where you can set how much stars will be active.

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,124 @@
...
+
+    &:hover {
+      border: 1px solid $button-color;
+      background-color: $back-color;
+      color: $button-color;
+    }
+  }
+}
+
+.stars {
```
Comment (Esceype) [inline src/styles/index.scss:104]: Stars block should be in a separate file

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,124 @@
-body {
+$main-text-color: #060B35;
+$second-text-color: #616070;
+$button-color: #00ACDC;
+$back-color: #FFF;
```
Comment (Esceype) [inline src/styles/index.scss:4]: All variables should be in a separate file

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,124 @@
...
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```
Comment (Esceype) [inline src/styles/index.scss:9]: Add the fallback font

**Review by Esceype — CHANGES_REQUESTED** [review]
Well done!
Please read the following comments and pay attention to stars block, check the requirements again for this and to the previous one task

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**